### PR TITLE
Document how to manage resources for MVP

### LINKED ISSUE #2620: 📝 Document flexible resources
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/overview/pricing.html
https://docs.platform.sh/increase-observability/metrics.html
And elsewhere

### What exactly should be updated?

The upcoming release of flexible resources means how we document plans and resource usage. 

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Closes #2620 

## What's changed

Added section/page about how users can manage resource allocation during the MVP phase.

### CODE REVIEW

File: sites/friday/src/manage-resources/_index.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+By default, {{% vendor/name %}} allocates a container profile to each app and service depending on the range of resources it's expected to need.
+
+Each container profile gives you access to a specific list of CPU and RAM combinations.
+Using the {{% vendor/name %}} CLI or Console, you can then pick a CPU and RAM combination for each of your apps and services.
+
+There are four container profiles available: `high_cpu`, `balanced`, `high_memory`, and `higher_memory`.
+The following table displays the different CPU and RAM combinations each container profile provides:
+
+| CPU  | `high_cpu`   | `balanced` | `high_memory` | `higher_memory` |
```
Comment (chadwcarlson) [inline sites/friday/src/manage-resources/_index.md:96]: These are all caps, no?

File: sites/friday/src/manage-resources/_index.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+## Configure individual container resources
+
+{{< codetabs >}}
+
++++
+title= Using the CLI
++++
+
+Run the `resources: set` command, and follow the prompts to set resources for each of your apps and services.
```
Comment (flovntp) [inline sites/friday/src/manage-resources/_index.md:49]: `resources:set` with no space ;)

Comment (chadwcarlson) [inline sites/friday/src/manage-resources/_index.md:49]:
```suggestion
Run the `resources:set` command, and follow the prompts to set resources for each of your apps and services.
```

File: sites/friday/src/manage-resources/_index.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+{{< note title="Tip" >}}
+
+You can keep an eye on your costs in the {{% vendor/name %}} Console:
+
+- To view a monthly estimate of how much each of your projects is expected to cost,
+navigate to your organization and open the **Overview** tab.
+
+- To view a monthly estimate detailing the resources allocated to each instance in a project and the related costs,
+navigate to your project and open the **Overview** tab.
```
Comment (chadwcarlson) [inline sites/friday/src/manage-resources/_index.md:78]: I'm confused by the location of the "Overview" tab for these two. I can find Overview under a particular environment, but neither under a Project nor an Organization.

**Comment by chadwcarlson** [comment]
https://docs.upsun.com.manage-resources-z2qlfya-652soceglkw4u.eu-3.platformsh.site/manage-resources.html

File: sites/friday/src/manage-resources/_index.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+  - "flexible resources"
+  - "CPU"
+  - "RAM"
+  - "disk storage"
+  - "horizontal scaling"
+  - "scaling"
+---
+
+{{% vendor/name %}} allows you to configure project resources and adjust them at any time.
+On each of your environments, you can define which amount of CPU, RAM,
```
Comment (chadwcarlson) [inline sites/friday/src/manage-resources/_index.md:16]:
```suggestion
On each of your environments, you can define how much CPU, RAM,
```

File: sites/friday/src/manage-resources/_index.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
+   ![Apps and services tree](/images/flexible-resources/apps-services-tree.png "0.2")
+   A window pops up.
+3. Click **Configure**.
+4. Select values in the dropdown menus depending on your needs:
+   ![Configure your resources on the current environment window](/images/flexible-resources/configure-flexible-resources.png)
+   Note that the values available in the **CPU & RAM** menus depend on the [container profile](#1-configure-your-container-profiles) of each instance.
```
Comment (chadwcarlson) [inline sites/friday/src/manage-resources/_index.md:70]:
```suggestion
   Note that the values available in the **CPU & RAM** menus depend on the [container profile](#advanced-container-profiles) of each instance.
```

File: sites/friday/src/manage-resources/_index.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+
+In most cases, it's best not to adjust container profiles.
+As a best practice, {{% vendor/name %}} recommends adjusting the profile of a container **only if the CPU/RAM ratio doesn’t match how the container scales**,
+taking into account both production and preview (staging and development) environments.
+
+To adjust a container profile, amend the value of the `container_profile` key in your configuration:
+
+```yaml {configFile="app"}
+container_profile: HIGH_MEMORY
+```
```
Comment (chadwcarlson) [inline sites/friday/src/manage-resources/_index.md:167]:
```suggestion
```yaml {configFile="app"}
applications:
    APP_NAME:
        container_profile: HIGH_MEMORY
```
```

File: sites/friday/src/manage-resources/_index.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+   ![Configure your resources on the current environment window](/images/flexible-resources/configure-flexible-resources.png)
+   Note that the values available in the **CPU & RAM** menus depend on the [container profile](#advanced-container-profiles) of each instance.
+   - For each app and service displayed, select a CPU & RAM combination and enter the amount of disk/storage you want to allocate to each container.
+   - For each of your apps and workers, select the number of instances you want to deploy.
+5. Click **Save**.</br>
+   You environment is redeployed.
+
+{{< /codetabs >}}
+
+## Keep an eye on your costs
```
[the remark this answers is not in the document]
Comment (AnouckColson) [reply sites/friday/src/manage-resources/_index.md:78]: I think it's important info to share. I've asked Niti to confirm which access rights may be needed.

**Review by chadwcarlson — DISMISSED** [review]

File: sites/friday/src/manage-resources/_index.md [hunk 8]
```diff
@@ -0,0 +1,279 @@
...
+For apps and workers, you can also define how many instances you want to deploy.
+To do so, follow these steps:
+
+{{< codetabs >}}
+
++++
+title= Using the CLI
++++
+
+To define how may instances of an app or worker you want to deploy,
```
Comment (nicogommen) [inline sites/friday/src/manage-resources/_index.md:98]: Small typo here, I believe `may` should be `many`.

Comment (chadwcarlson) [inline sites/friday/src/manage-resources/_index.md:98]:
```suggestion
To define how many instances of an app or worker you want to deploy,
```

Comment (chadwcarlson) [reply sites/friday/src/manage-resources/_index.md]: Thanks!

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**